plugin.rb

```diff
@@ -12,27 +12,22 @@
 module ::OpencollectivePlugin
   BADGE_NAME ||= 'OpenCollective Donator'.freeze
 
-  def self.get_data!
+  def self.badges_grant!
     token=SiteSetting.opencollective_access_token
     collective=SiteSetting.opencollective_collective_name
 
     if token=="" or collective==""
       puts "Fetching users from opencollective failed!"
       puts "Please configure settings.yml for the opencollective plugin"
-      ::PluginStore.set('discourse-opencollective-plugin','user_data', nil)
       return
     end
+
     conn = Faraday.new(url: 'https://opencollective.com',
                        headers: { 'Authorization' => "Bearer #{token}" })
 
     response = conn.get "/api/groups/#{collective}/users"
     data = JSON.parse response.body
```

Comment:
Hi @atomaka ,
I think this variable has to be renamed as
retrieve = JSON.parse response.body